src/aiortc/contrib/media.py

```diff
@@ -431,6 +432,8 @@ def __init__(self, file, format=None, options={}):
         self.__tracks = {}
         self.frame_height = None
         self.frame_width = None
+        self.keypoints_queue = asyncio.Queue()
```

Comment:
Use self.__keypoints_queue to be consistent with naming convention. Same goes for self.__frame_height and width if not previously defined

Reply:
Done